lib/PPI/Statement/Include.pm

```diff
@@ -251,9 +254,12 @@ sub feature_mods {
 		## tried using feature.pm, but it is impossible to install future
 		## versions of it, so e.g. a 5.20 install cannot know about
 		## 5.36 features
+
+		# crude proof of concept hack due to above
+		return { signatures => 1 } if version::parse($perl_version) >= 5.035;
```

Comment:
this should be some kind of configuration, ideally supported by user config file

Reply:
User-configurable recognition is already a thing via _custom_feature_include_cb and _custom_feature_includes. Additionally i have a todo for that here: https://github.com/Perl-Critic/PPI/pull/280#issuecomment-2493357524